features/crc/soft/FastCRC.h

```diff
@@ -0,0 +1,202 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MBED_FAST_CRC_H
+#define MBED_FAST_CRC_H
+
+#include "mbed_crc.h"
+#include "mbed.h"
```

Comment:
This should not be included within our code base (include everything is not an answer to what symbol is missing), please include what is needed here.

Reply:
Good catch 👍 

Sounds like we should have a script to catch these... 🤔